build.gradle

```diff
@@ -195,9 +195,12 @@ test {
     String CI = "$System.env.CI"
     useTestNG{
         if ( "$System.env.CLOUD" =="true"){
-            includeGroups 'cloud', 'bucket'
+            // run everything
+            includeGroups 'cloud', 'bucket', 'cloud_todo', 'bucket_todo'
         } else {
-            excludeGroups 'cloud', 'bucket'
+            // run everything except the "todo" tests
+            excludeGroups 'cloud_todo', 'bucket_todo'
+            includeGroups 'cloud', 'bucket'
```

Comment:
Include excludes anything that isn't explicitly included.  So you're accidentally excluding all tests that are not cloud or bucket.   If you look at the [test log](https://storage.googleapis.com/hellbender/test/build_reports/1840.2/tests/index.html) you'll see that you're actually not running any tests now in the standard run.